finagle-core/src/main/scala/com/twitter/finagle/param/Params.scala

```diff
@@ -128,9 +128,16 @@ object HighResTimer {
    *
    * It is a shared resource and as such, `stop` is ignored.
    */
-  val Default: com.twitter.util.Timer =
+  private[finagle] val Default: com.twitter.util.Timer =
```

Comment:
I don't think we can reduce the visibility on this Timer, people use it all the time.

Reply:
OK